scripts/generator/templates/template-vaadin-gradle-plugin-pom.xml

```diff
@@ -113,6 +118,86 @@
                 </executions>
             </plugin>
 
+            <plugin>
+                <groupId>com.coderplus.maven.plugins</groupId>
+                <artifactId>copy-rename-maven-plugin</artifactId>
+                <version>1.0.1</version>
+                <executions>
+                    <execution>
+                        <id>rename-hilla-plugin-to-vaadin</id>
+                        <phase>process-sources</phase>
+                        <goals>
+                            <goal>rename</goal>
+                        </goals>
+                        <configuration>
+                            <sourceFile>src/main/kotlin/com/vaadin/gradle/plugin/HillaPlugin.kt</sourceFile>
+                            <destinationFile>src/main/kotlin/com/vaadin/gradle/plugin/VaadinPlugin.kt</destinationFile>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <version>3.0.2</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.apache.groovy</groupId>
+                        <artifactId>groovy-all</artifactId>
+                        <version>4.0.18</version>
+                        <type>pom</type>
+                    </dependency>
+                </dependencies>
+                <executions>
+                    <execution>
+                        <id>replace-package-declaration-in-src-files</id>
+                        <phase>process-sources</phase>
+                        <goals>
+                            <goal>execute</goal>
+                        </goals>
+                        <configuration>
+                            <scripts>
+                                <script>
+                                    <![CDATA[
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.nio.file.Path
+import java.util.stream.Collectors
+
+// Define the directory to search for Kotlin files
+String directoryPath = 'src/main/kotlin/com/vaadin/gradle/plugin'
```

Comment:
it seems this path is the reason that failed the build.
if i update it as `vaadin-gradle-plugin/src/main/kotlin/com/vaadin/gradle/plugin`.. the build works..

if i dont have the `vaadin-gradle-plugin/` , running from the `gradle-plugin` module works.

seems here should use a relative path instead

Reply:
validated using this PR https://github.com/vaadin/platform/pull/4987.
it works..